docs/excel/excel-add-ins-tables.md

```diff
@@ -353,6 +353,35 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+### AutoFilter
+
+> [!NOTE]
+> `Table.autoFilter` is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+As of the `ExcelApi 1.9` requirement set, add-ins have access to the table's [AutoFilter](/javascript/api/excel/excel.autofilter). An AutoFilter object is the entire filter structure of a table or range. All the aforementioned filter operations are compatible with the auto-filter. The single access point does make it easier to access and manage multiple filters.
```

Comment:
Consider changing "All the aforementioned filter operations..." -> "All of the filter operations described previously in this article..."    (although wordier, likely better for localization)